data/autoyast_sle15/create_hdd/create_hdd_maintenance.xml.ep

```diff
@@ -135,14 +135,32 @@
         <products config:type="list">
             <product><%= uc $get_var->('SLE_PRODUCT') %></product>
         </products>
-        % if ($get_var->('SCC_ADDONS') =~ m/\brt\b/) {
-        <kernel>kernel-rt</kernel>
-        % }
         <patterns config:type="list">
             % for my $pattern (@$patterns) {
             <pattern><%= $pattern %></pattern>
             % }
         </patterns>
+        <!-- workaround for bsc#1202234 -->
+        <packages config:type="list">
+        % my %product_list = (
```

Comment:
apparently we don't need the mapping, could you please check line 13?

Reply:
Thanks, it was not needed indeed...